php/Terminus/Models/Environment.php

```diff
@@ -775,21 +763,4 @@ private function getSettings($setting = null) {
 
   }
 
-  /**
-   * Changes the environment's settings
-   *
-   * @param array $settings Key/value pairs to set in the environment settings
-   * @return bool
-   */
-  private function updateSetting(array $settings = []) {
```

Comment:
I think we're moving away from directly setting these variables altogether. If any are still in use then they probably need to be moved to panthon.yml. I don't know if any plugins are relying on this function though.